packages/frontend/package.json

```diff
@@ -15,7 +15,7 @@
         "@lightdash/common": "^0.251.2",
         "@sentry/react": "^6.17.6",
         "@sentry/tracing": "^6.17.6",
-        "@tanstack/react-table": "^8.5.11",
+        "@tanstack/react-table": "^8.5.13",
```

Comment:
what'st he reason for the upgrade ?

Reply:
I'll revert this. I had a really weird bug and thought that would help.